HW_6/HW_7.py

```diff
@@ -0,0 +1,248 @@
+
+
+class Article:
+    def __init__(self, header, text):
+        self.header = header
+        self.text = text
+
+    def get_header():
+        input_versions = ['News', 'PrivatAdv', 'Consider']
+        while True:
+            header = input(f"Enter the topic you are interested in: {', '.join(input_versions)}\n")
+            if header not in input_versions:
+                print('Please select correct topic')
+            else:
+                break
+        return header
+
+    def get_text():
+        print("Type your text here. Press Enter twice to exit from text adding mode")
+        input_text = ''
+        while True:
+            new_line = input()
+            if new_line != '':
+                input_text += new_line + '\n'
+            else:
+                input_text = input_text[:-1]
+                break
+        text = normalization.normalization(input_text)
+        return text
+
+    def get_file_choise():
+        versions = ['json', 'txt', 'xml']
+        while True:
+            file_choise = input('Choose file type you want to use? Answer: json/txt/xml\n:')
+            if file_choise not in versions:
+                print('Please select correct file type')
+            else:
+                break
+        return file_choise
+
+    def get_default_file( header, file_choise):
+        global default_file
+        if file_choise == 'json':
+            if header == 'News':
+                default_file = 'default_news.json'
+            elif header == 'PrivatAdv':
+                default_file = 'default_advs.json'
+            elif header == 'Consider':
+                default_file = 'default_considers.json'
+        elif file_choise == 'txt':
+            if header == 'News':
+                default_file = 'default_news.txt'
+            elif header == 'PrivatAdv':
+                default_file = 'default_advs.txt'
+            elif header == 'Consider':
+                default_file = 'default_considers.txt'
+        elif file_choise == 'xml':
+            if header == 'News':
+                default_file = 'default_news.xml'
+            elif header == 'PrivatAdv':
+                default_file = 'default_advs.xml'
+            elif header == 'Consider':
+                default_file = 'default_considers.xml'
+        return default_file
+
+
+    def check_path(default_file):
+        user_input = input("Enter the path of your file: ")
+        if os.path.exists(user_input):
+            try:
+                file_to_parse = open(user_input, 'r+')
+            except:
+                print("File is not valid. I'll use mine ")
+                file_to_parse = open(default_file, 'r+')
+        else:
+            print("I did not find the file at, " + str(user_input) + " I'll use mine ")
+            file_to_parse = open(default_file, 'r+')
+        return file_to_parse
+
+
+    def parse_file(file_choise, file_to_parse):
+        text_list = {}
+        if file_choise == 'json':
```

Comment:
it'd be better to create a separate functions "parse_json", "parse_txt", "parse_xml" here